config/README.md

```diff
@@ -18,7 +18,7 @@ Adjusting `global` settings ensures a non-interfering setup.
 
 ### Global
 
- * `global.root_path` - [string]: Defines the path for all Iceoryx2 files and directories.
+ * `global.root_path_{unix|windows}` - [string]: Defines the path for all Iceoryx2 files and directories.
```

Comment:
I guess there is a technical reason why it cannot just be `root_path`.

Btw, for config files it is nicer to have kebab-case for the variable names. You just need to use `#[serde(rename_all = "kebab-case")]`. Just in case you want to change it in a future version. We are still quite new with iceoryx2 so we still have the chance without breaking too much. I could take care of that if you like the idea.

Reply:
@elBoberido I like the idea, please take care of it :smile: